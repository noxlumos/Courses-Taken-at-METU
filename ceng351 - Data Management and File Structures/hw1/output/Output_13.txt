
--------------------------------------------------------------
*** Q8 ***
isbn	book_name	rating
Q8: Exception occured: 

java.lang.NullPointerException

--------------------------------------------------------------
