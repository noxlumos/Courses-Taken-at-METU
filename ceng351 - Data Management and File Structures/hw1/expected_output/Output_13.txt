
--------------------------------------------------------------
*** Q8 ***
isbn	book_name	rating
4444444444444	Temp4	3.0
5555555555555	Temp4	3.0
9786059389754	Viyana Nokta	3.0

--------------------------------------------------------------
